journal/modernized/05-chapter-5.adoc

```diff
@@ -360,10 +363,14 @@ David Ferris
 [.signed-section-context-close]
 Wilmington, 5th Mo. 1779.
 
+[.postscript]
```

Comment:
This can stay, because `[.postscript]` tags still work outside of embedded sections.👍🏻